.github/workflows/build/action.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Build SWC'
```

Comment:
Move existing SWC build job into a composite action, so it can be used in the existing workflow and in the new e2e workflow.